content/en/docs/setup/install/operator/index.md

```diff
@@ -75,11 +73,197 @@ accessible to `istioctl` by using this command:
 $ istioctl experimental profile list
 {{< /text >}}
 
-Optionally, you can use the `-s` flag with a install package path to see the
-list of configuration profiles available for other Istio versions:
+## Customize Istio settings using the IstioControlPlane API
+
+You can change a feature or component setting by using the [IstioControlPlane API](https://github.com/istio/operator/blob/release-1.3/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto).
```

Comment:
would be nice if we could sort out how to render this somehow, as I think @geeknoid is working on.

Reply:
Yes, that's been fixed and there's the ptr to the rendered proto PR:
https://github.com/istio/istio.io/pull/4948
Once that's approved we should add a link here to that doc, although imo we should retain the link to the proto too, since that has not been reshuffled to be in alphabetical order and is easier to read.

Reply:
Ok, proto PR is merged and I've updated the link - ptal @sdake.